.travis.yml

```diff
@@ -0,0 +1,209 @@
+os: linux
+dist: xenial
+language: java
+jdk:
+  - openjdk11
+  
+addons:
+  firefox: "76.0.1"
+  
+services:
+  - xvfb
+  - docker
+
+  
+cache:
+  directories:
+    - $HOME/.m2
+# the cache can grow constantly
+before_cache:
+  - rm -rf $HOME/.m2/repository/org/alfresco/alfresco-share*
+  
+branches:
+  only:
+    - master
+    - /support\/.*/
+    - fix/Travis-share-build
+    
+stages:
+  - build
+  - tests
+    if: commit_message !~ /\[skip-tests\]/
+  - release
+
+
+# This should not be required on community build
+before_install:
+  - "cp _ci/.travis.settings.xml $HOME/.m2/settings.xml"
+
+  
+jobs:
+  include:
+    - name: "Build"
+      stage: build
+      
+    - name: "WhiteSource scan"
+      stage: build
+      # only on SP branches or master and if it is not a PR
+      if: fork = false AND (branch = develop OR branch =~ /support\/SP\/.*/ OR branch =~ /^APPS-.*$/) AND type != pull_request
+      script:
+        # Download the latest version of WhiteSource Unified Agent
+        - curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+        # Run WhiteSource Unified Agent
+        - java -jar wss-unified-agent.jar -apiKey ${WHITESOURCE_API_KEY} -c _ci/.wss-unified-agent.config 
+        
+    - name: "Source Clear Scan (SCA)"
+      stage: build
+      # only on SP branches or master and if it is not a PR
+      if: fork = false AND (branch = develop OR branch =~ /support\/SP\/.*/ OR branch =~ /^APPS-.*$/) AND type != pull_request
+      # Run Veracode
+      install: skip
+      script: travis_wait 30 bash _ci/source_clear.sh
+      
+    - name: "Push Image To Quay"
+      stage: build
+  
+    - name: "SHARE PO ADMIN-TOOLS"
+      stage: tests
+      jdk: openjdk11
+      install:
```

Comment:
There are quite a few repetitive sections following. Please check if it's possible to rewrite using YAML anchors and aliases: https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/